app/styles/page-not-found.css

```diff
@@ -0,0 +1,13 @@
+.not-found-container{
+  margin: auto;
+  text-align: center;
+}
+
+.not-found-img {
+  width: 70%;
```

Comment:
The image seems too big in the screenshot according to other elements. Can we please have an image of height around `400px` or less and accordingly the width of the image.
Also, ensure that it's responsive for all devices, text and the image